arch/arm/src/imxrt/imxrt_usbdev.c

```diff
@@ -871,7 +872,7 @@ static inline void imxrt_set_address(struct imxrt_usbdev_s *priv,
 
   imxrt_chgbits(USBDEV_DEVICEADDR_MASK,
                 priv->paddr << USBDEV_DEVICEADDR_SHIFT,
-                IMXRT_USBDEV_DEVICEADDR);
+                IMXRT_USBDEV_DEVICEADDR(0));
```

Comment:
Maybe I don't understand correctly, but doesn't this implementation now only use USB0? Should the register calls be something like `IMXRT_USBDEV_DEVICEADDR(priv->usbno)` or something?